### PR TITLE
(DOCSP-9184): Port GraphQL Connect

### PR DESCRIPTION
**Pull request by nathan-contino-mongo**
JIRA: https://jira.mongodb.org/browse/DOCSP-9184

Stage: https://docs-mongodbcom-staging.corp.mongodb.com/realm/pegasus/DOCSP-9184/graphql/connect.html

### CODE REVIEW

File: source/graphql/connect.txt [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+.. tabs::
+
+   .. tab:: Apollo (React)
+      :tabid: apollo-react
+
+      Apollo provides the :npm:`@apollo/react-hooks
+      <packages/@apollo/react-hooks>` package which contains a set of
+      React hooks that you can use to execute GraphQL operations. The
+      package also contains an ``ApolloProvider`` component that make
+      your ``ApolloClient`` available to all components in your app.
```
Comment (mongomoe) [inline source/graphql/connect.txt]: Should "make" here be "makes" 

"The package also contains an ``ApolloProvider`` component that makes your ``ApolloClient`` available to all components in your app."

File: source/graphql/connect.txt [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+
+Add the GraphQL Client to Your App
+----------------------------------
+
+.. tabs::
+
+   .. tab:: Apollo (React)
+      :tabid: apollo-react
+
+      Apollo provides the :npm:`@apollo/react-hooks
```
Comment (mongomoe) [inline source/graphql/connect.txt]: "Apollo provides the @apollo/react-hooks package which contains a set of React hooks that you can use to execute GraphQL operations."

should be:

"Apollo provides the @apollo/react-hooks package, which contains a set of React hooks that you can use to execute GraphQL operations."

Comment (nathan-contino-mongo) [reply source/graphql/connect.txt]: While I do generally agree that a comma should usually precede "which", upon reconsideration I believe that this is a situation where no comma and "that" is more appropriate (at least by American english standards). So I'm doing that instead. Good catch though!

File: source/graphql/connect.txt [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+   3. :doc:`Generate a collection schema
+      </mongodb/enforce-a-document-schema>` for the
+      ``sample_mflix.movies`` collection. Add a root-level ``title``
+      field to the schema with the value set to ``"Movie"``.
+
+   4. Enable :doc:`anonymous authentication </authentication/anonymous>`.
+
+   5. Deploy your changes.
+
+   Once your app is set up, paste your App ID into the CodeSandbox to
```
Comment (mongomoe) [inline source/graphql/connect.txt]: "Once your app is set up" is written in the passive voice and may read better with active voice:
The following is a suggested variation that get's the same message across: 
```
// suggested sentence:
Once you set up your app, paste your App ID into the CodeSandbox to connect and use the GraphQL API.
```

Comment (nathan-contino-mongo) [reply source/graphql/connect.txt]: Definitely agree that we can improve this wording. Thanks for the suggestion!

File: source/graphql/connect.txt [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+
+      Wrap your React application with an ``ApolloProvider`` and pass
+      the ``client`` that you created to the provider as a prop. Any
+      components rendered inside of the ``ApolloProvider`` can use
+      Apollo React hooks.
+
+      .. code-block:: javascript
+         :emphasize-lines: 1, 6
+
+         import { ApolloProvider } from "@apollo/react-hooks";
```
Comment (mongomoe) [inline source/graphql/connect.txt]: I wonder why the "import" word and "<package-name>" is not colored in this code-block but is colored in the other code-block.

Comment (nathan-contino-mongo) [reply source/graphql/connect.txt]: Good question. I tried playing around with it and I have no idea. One of the great mysteries of life I suppose.

**Pull request merged**